plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -121,11 +121,13 @@ public DockerInstanceProvider(DockerConnector docker,
         this.supportedRecipeTypes = Collections.singleton("dockerfile");
         this.projectFolderPath = projectFolderPath;
 
+        allMachinesSystemVolumes = filterEmptyAndNullValues(allMachinesSystemVolumes);
+        devMachineSystemVolumes = filterEmptyAndNullValues(devMachineSystemVolumes);
         if (SystemInfo.isWindows()) {
-            allMachinesSystemVolumes = escapePaths(allMachinesSystemVolumes);
-            devMachineSystemVolumes = escapePaths(devMachineSystemVolumes);
+            allMachinesSystemVolumes = escapePaths(filterEmptyAndNullValues(allMachinesSystemVolumes));
```

Comment:
Why did you add `filterEmptyAndNullValues` to the `escapePaths` method, as far as i understand they are already filtered 2 lines above + i think that it is better to name method `removeEmptyAndNullValues` which is more obvious against filter.